load_test.py

```diff
@@ -0,0 +1,96 @@
+import time
+from sdcm.tester import ClusterTester
+from sdcm.db_stats import PrometheusDBStats
+
+
+class LoadTest(ClusterTester):
+    """
+    Test Scylla load with cassandra-stress
+    """
+
+    def run_load_and_check_for_admission_control(self, prepare_base_cmd, stress_base_cmd, job_num=8):  # pylint: disable=invalid-name
+        # pylint: disable=too-many-locals
+
+        prometheus = PrometheusDBStats(self.monitors.nodes[0].public_ip_address)
```

Comment:
these can be defined outside and passed to the relevant function

Reply:
and the below:

Reply:
done